charts/microgateway/README.md

```diff
@@ -99,117 +99,117 @@ This chapter provides a simple example to help you get the Airlock Microgateway
-| config.passphrase.secretName | string | "" | Name of an existing secret containing: <br> <br> passphrase: `passphrase` |
-| config.passphrase.useExistingSecret | bool | `false` | Specifies whether a pre-existing secret should be mounted. |
-| config.passphrase.value | string | "" | The passhprase which will be stored and used in a secret. |
-| config.tlsSecretName | string | "" | Name of an existing secret containing:<br><br> _Virtual Host:_<br> Certificate: `frontend-server.crt`<br> Private key: `frontend-server.key`<br> CA: `frontend-server-ca.crt` <br> :exclamation: Update `route.tls.destinationCACertificate` accordingly.<br><br> _Backend:_<br> Certificate: `backend-client.crt`<br> Private key: `backend-client.key`<br> CA: `backend-server-validation-ca.crt` |
-| echo-server | object | See `echo-server.*`: | Pre-configured [Echo-Server](#echo-server). |
-| echo-server.enabled | bool | `false` | Deploy pre-configured [Echo-Server](#echo-server). |
-| extraVolumeMounts | list | `[]` | Add additional volume mounts. |
-| extraVolumes | list | `[]` | Add additional volumes. [Volumes](https://kubernetes.io/docs/concepts/storage/volumes/) |
-| fullnameOverride | string | `""` | Provide a name to substitute for the full names of resources. |
-| hpa | object | See `hpa.*`: | [Horizontal Pod Autoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) to scale <br> Microgateway based on Memory and CPU consumption.<br><br> :exclamation: Check [API versioning](https://kubernetes.io/docs/concepts/overview/kubernetes-api/#api-versioning) when using this Beta feature. |
-| hpa.enabled | bool | `false` | Deploy a horizontal pod autoscaler. |
-| hpa.maxReplicas | int | `10` | Maximum number of Microgateway replicas. |
-| hpa.minReplicas | int | `1` | Minimum number of Microgateway replicas. |
-| hpa.resource.cpu | int | `50` | Average Microgateway CPU consumption in percentage to scale up/down.<br><br> :exclamation: Please set the resource request parameter `resources.cpu` to a value reflecting your actual resource needs if you use autoscaling based on cpu consumption. Otherwise autoscaling will not work as expected. |
-| hpa.resource.memory | string | `"3Gi"` | Average Microgateway Memory consumption to scale up/down.<br><br> :exclamation: Update this setting depending on your `resources.limits.memory` setting. |
-| image.pullPolicy | string | `"IfNotPresent"` | Pull policy (`Always`, `IfNotPresent`, `Never`) |
-| image.repository | object | "See `image.repository.*`" | Image repositories for the Airlock Microgateway. |
+| Key | Type | Default                                                | Description |
```

Comment:
Ist der Change mit den zusätzlichen Whitespaces für das Aligning in der Tabelle manuel oder generiert? Intended auf jedenfall nehme ich an?